docs/how-tos/configure-teleop-workspace.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Configure a teleop workspace"
+linkTitle: "Configure a teleop workspace"
+weight: 10
+type: "docs"
+description: "Create and configure a teleop workspace with widgets."
+images: ["/how-tos/teleop/full-workspace.png"]
+icon: true
+tags: ["teleop", "configuration"]
+languages: []
+viamresources: ["sensor", "camera", "movement sensor"]
+platformarea: ["viz", "data"]
+level: "Intermediate"
+date: "2024-11-13"
+# updated: "2024-08-26"  # When the tutorial was last entirely checked
+cost: "0"
+---
+
+You can use teleop to create a custom workspace where you can visualize data from a machine.
+Follow this guide to learn how to create a custom workspace using teleop, visualizing data from a camera, a sensor, or a movement sensor.
+
+{{% alert title="In this page" color="info" %}}
+
+- [Configure a workspace](#configure-a-workspace)
+
+{{% /alert %}}
+
+## Prerequisites
+
+{{% expand "A configured machine with teleoperable components" %}}
+
+Make sure your machine has at least one of the following:
+
+- A [camera](/components/camera/)
+- A [movement sensor](/components/movement-sensor/)
+- A [sensor](/components/sensor/)
+
+See [configure a machine](/how-tos/configure/) for more information.
```

Comment:
figure this covers device requirements and viam app requirements? lmk and I can add more but this isn't a super basic thing to do with viam so I figured a tiny bit of cross-linking was ok

Reply:
yep, I like it